musicbrainz/csv/convert_to_csv.py

```diff
@@ -0,0 +1,101 @@
+# This file converts a JSON Lines file from MusicBrainz to a CSV file
+# For OpenRefine Reconciling
+# Input file can be downloaded in
+# https://data.metabrainz.org/pub/musicbrainz/data/json-dumps/20240522-001002/
+# Output file can be imported to OpenRefine
+import json
+import copy
+
+header = ["id"]
+values = []
+
+with open("test2", "r") as f:
+    json_data = [json.loads(m) for m in f]
+
+# print(json_data)
+
+
+def extract(data, first_level: bool = True, key: str = "", value: dict = {}):
```

Comment:
I don't know what editor you're using, but whatever it is, you need to configure it to flag things like this. Unless you know what you're doing, you don't use mutable data structures as default values.

https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil